src/main/java/com/minecolonies/items/ItemSupplyChestDeployer.java

```diff
@@ -154,12 +156,12 @@ else if (check(world, pos.north(), false, false))
      */
     private void spawnShip(@NotNull World world, @NotNull BlockPos pos, EntityPlayer entityPlayer, @NotNull EnumFacing chestFacing)
     {
-        world.setBlockState(pos.up(), Blocks.chest.getDefaultState().withProperty(BlockChest.FACING, chestFacing));
+        world.setBlockState(pos.up(), Blocks.CHEST.getDefaultState().withProperty(BlockChest.FACING, chestFacing));
 
         placeSupplyShip(world, pos, chestFacing);
 
         fillChest((TileEntityChest) world.getTileEntity(pos.up()));
-        PlayerProperties.get(entityPlayer).placeSupplyChest();
+        //PlayerProperties.get(entityPlayer).placeSupplyChest();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)